### PR TITLE
12.0 mig account_move_line_tax_editable

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

File: account_move_line_tax_editable/models/account_move_line.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        string="Is tax data editable?", compute='_compute_is_tax_editable')
+
+    @api.multi
+    @api.depends('move_id.state')
+    def _compute_is_tax_editable(self):
+        for rec in self:
+            rec.is_tax_editable = rec._get_is_tax_editable()
+
+    @api.multi
+    def _get_is_tax_editable(self):
```
Comment (pedrobaeza) [inline account_move_line_tax_editable/models/account_move_line.py]: Please remove this useless extra method and integrate it in main one, and include a test for this computed field.

Comment (ernestotejeda) [reply account_move_line_tax_editable/models/account_move_line.py]: Changes done

File: account_move_line_tax_editable/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Move Line Tax Editable',
+    'summary': """
+        Allows to edit taxes on non-posted account move lines""",
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://www.acsone.eu',
```
Comment (nikul-serpentcs) [inline account_move_line_tax_editable/__manifest__.py]: @ernestotejeda URL should be `https://github.com/OCA/account-financial-tools`

File: account_move_line_tax_editable/tests/test_account_move_line_tax_editable.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestAccountMoveLineTaxEditable, cls).setUpClass()
+
+        acc_obj = cls.env['account.account']
+        account100 = acc_obj.create({
+            'code': '100',
+            'name': 'Account 100',
+            'user_type_id': cls.env.ref(
```
Comment (nikul-serpentcs) [inline account_move_line_tax_editable/tests/test_account_move_line_tax_editable.py:18]: @ernestotejeda  If you need ID then use this `cls.env.('account.data_account_type_receivable')`

File: account_move_line_tax_editable/tests/test_account_move_line_tax_editable.py [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                'account.data_account_type_receivable').id,
+            'reconcile': True
+        })
+        account300 = acc_obj.create({
+            'code': '300',
+            'name': 'Account 300',
+            'user_type_id': cls.env.ref(
```
Comment (nikul-serpentcs) [inline account_move_line_tax_editable/tests/test_account_move_line_tax_editable.py:25]: same here

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
nitpicking

File: account_move_line_tax_editable/tests/test_account_move_line_tax_editable.py (hunk 3 above)
Comment (pedrobaeza) [reply account_move_line_tax_editable/tests/test_account_move_line_tax_editable.py:18]: That doesn't work on `setUpClass` method.

File: account_move_line_tax_editable/tests/test_account_move_line_tax_editable.py (hunk 4 above)
Comment (pedrobaeza) [reply account_move_line_tax_editable/tests/test_account_move_line_tax_editable.py:25]: The same.

File: account_move_line_tax_editable/views/account_move_line.xml [hunk 5]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="account_move_line_form_view">
```
Comment (tbaden) [inline account_move_line_tax_editable/views/account_move_line.xml]: cosmetical change (optional):
id before model
and
>  In the case where an inherited view has a name which does not follow the guidelines set above, prefer naming the inherited view after the original over using a name which follows the guidelines.

File: account_move_line_tax_editable/views/account_move.xml [hunk 6]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="ir.ui.view" id="account_move_form_view">
```
Comment (tbaden) [inline account_move_line_tax_editable/views/account_move.xml]: cosmetical change (optional):
id before model
and
>  In the case where an inherited view has a name which does not follow the guidelines set above, prefer naming the inherited view after the original over using a name which follows the guidelines.

**Review by tbaden — COMMENTED** [review]
https://github.com/OCA/account-financial-tools/pull/812#discussion_r278805180
and  some cosmetical changes, but these are optional

**Comment by nikul-serpentcs** [comment]
> [#812 (comment)](https://github.com/OCA/account-financial-tools/pull/812#discussion_r278805180)
> and some cosmetical changes, but these are optional

@tbaden   It's Not optional, it's mandatory to change URL as per OCA stander.

**Comment by tbaden** [comment]
@nikul-serpentcs yes your change is mandatory, i mean mine as optional

**Review by pedrobaeza — APPROVED** [review]
Apart from the other comments

**Review by tbaden — APPROVED** [review]
thanks for the optional changes.

@nikul-serpentcs your changes were made as well, anything else or good to go?

**Comment by pedrobaeza** [comment]
I'm going to merge as the comments have been honored

**Pull request merged**